source/fundamentals/indexes.txt

```diff
@@ -1,35 +1,60 @@
-.. uses geo.rst
+.. _pymongo-indexes:
 
-Indexing
+Indexes
+=======
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
```

Comment:
I think this should probably be reference, since there are no steps (I think that was the criteria Sarah gave for tuts?)